components/compliance-service/reporting/relaxting/profiles.go

```diff
@@ -88,10 +88,10 @@ func (esprofile *ESInspecProfile) parseInspecProfile(profile inspec.Profile) err
 	esprofile.Sha256 = profile.Sha256
 	// no need for Status and SkipMessage here as it's not static metadata of the profile
 	var groups []inspec.Group
-	for _, group := range groups {
+	for _, group := range profile.Groups {
 		eGroup := inspec.Group{
-			ID:       group.ID,
-			Title:    group.Title,
+			ID:       group.Id,
+			Title:    &group.Title,
```

Comment:
I don't think taking a reference to the loop temporary is safe here.

Consider the example here https://play.golang.org/p/rb-sdxW-64v

Rather I think you'll need something like:

```golang
for _, group := range profile.Groups {
        title := group.Title
	eGroup := inspec.Group{
		ID:       group.Id,
		Title:    &title,
		Controls: group.Controls,
	}
}
```

Reply:
profile.Groups is actually []*Group, so the the example would be more accurately represented as:
https://play.golang.org/p/BcU7Af4Tdut

Reply:
I see! Cool, this is fine then

Reply:
cool. thanks!